packages/nvidia_nat_app/src/nat_app/executors/runner.py

```
@@ -0,0 +1,221 @@
+            task.cancel()
+        actually_cancelled.add(name)
+
+    await asyncio.sleep(0)
+
+    for name in actually_cancelled:
+        task = target_tasks[name]
+        if not task.done():
+            try:
+                await task
+            except (asyncio.CancelledError, Exception):
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)